arkouda/pdarrayclass.py

```diff
@@ -65,6 +65,30 @@ def unescape(s):
         raise ValueError(("unsupported value from server {} {}".\
                               format(mydtype.name, value)))
 
+
+
+
+@typechecked
+def parse_single_int_array_value(msg : str) -> object:
+    """
+    Attempt to convert a scalar return value from the arkouda server to a
+    numpy string in Python. The user should not call this function directly. 
+    
+    Parameters
+    ----------
+    msg : str
+        scalar value in string form to be converted to a numpy string
+
+    Returns
+    -------
+    object numpy scalar         
+    """
+    fields = msg.split(" ",1)
+    dtname=fields[0]
+#    mydtype = dtype(dtname)
+    nfields = fields[1].split("\"")
```

Comment:
Recommend try..except block here in the event a server side error results in a msg that does not match the expected formatting

Reply:
done!